astro/src/content/quickstarts/quickstart-php-laravel-api.mdx

```diff
@@ -0,0 +1,424 @@
+- `panic`: Tellers may call this endpoint to call the police in case of an incident. The only valid role is `teller`.
+
+Both endpoints will be protected such that a valid JSON web token (JWT) will be required in the `app.at` cookie in order to be accessed. Additionally, the JWT must have a `roles` claim containing the appropriate role to use the endpoint.
+
+<Aside type="note">
+  If you simply want to run the application, there is a completed version in the `complete-application` directory. You can install [PHP](https://www.php.net) and [Composer](https://getcomposer.org/) and use the following commands to get it up and running if you do not want to create your own.
+
+  ```shell
+  cd complete-application
+  composer install
+  ./vendor/bin/sail up -d
+  ```
+
+  You can then follow the instructions in the "Run the API" section.
+</Aside>
+
+To create the application from scratch, you'll use [Laravel Sail](https://laravel.com/docs/10.x/installation#laravel-and-docker) to set up and run your application using Docker.
+
+```shell
+curl "https://laravel.build/fusionauth-quickstart-php-laravel-api?with=mariadb" | bash
+cd fusionauth-quickstart-php-laravel-api
+```
+
+### Initialize the Application
+
+Initialize the Laravel application using the following:
+
+```shell
+./vendor/bin/sail up
+```
+
+### Update Routes
+
+Change `routes/api.php` to add both `/make-change` and `/panic` endpoints for your ChangeBank application.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/routes/api.php"} lang="php"/>
+
+### Add Security
+
+Include FusionAuth's library to add JWT support to the authentication system.
+
+```shell
+./vendor/bin/sail composer require fusionauth/jwt-auth-webtoken-provider web-token/jwt-signature-algorithm-rsa
+```
+
+### Create Fields in the User Entity
+
+When installing Laravel, it will automatically make a migration to create a `users` table in the database. You'll have to edit the file `database/migrations/2014_10_12_000000_create_users_table.php` to change the default behavior and use UUID for the primary key, remove the need for a password and the remember token.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/database/migrations/2014_10_12_000000_create_users_table.php"} lang="php"/>
+
+You'll also need to edit `app/Models/User.php` to make the database changes consistent with your model.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/app/Models/User.php"} lang="php"/>
+
+You may remove all other migrations, as you won't need them.
+
+```shell
+rm database/migrations/2014_10_12_100000_create_password_reset_tokens_table.php
+rm database/migrations/2019_08_19_000000_create_failed_jobs_table.php
+rm database/migrations/2019_12_14_000001_create_personal_access_tokens_table.php
+```
+
+Run the remaining migrations to create the necessary tables in your database.
+
+```shell
+./vendor/bin/sail artisan migrate
```

Comment:
Got error:
![Screenshot 2023-10-24 at 4 12 17 PM](https://github.com/FusionAuth/fusionauth-site/assets/137117976/53ef70c3-4787-424e-b600-cef2c6de1d54)

Reply:
I've changed the order of the steps and it's now working.